health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/WorkflowCommentPopUp.js

```
@@ -41,35 +41,49 @@ const WorkflowCommentPopUp = ({ onClose, heading, submitLabel, url, requestPaylo
     const setCommentInPayloadForList = (payloadObject, path, comment) => {
         const keys = path.split(".");
+        const key = Object.keys(payloadObject)?.[0]; // Get the first key in the payloadObject
+        const value = payloadObject[key]; // Get the value associated with that key
+
+        // Check if the value is an array or a single object
+        if (Array.isArray(value)) {
+            // It's an array, so map through each item as before
+            return {
+                ...payloadObject,
+                [key]: value.map(item => {
+                    let updatedItem = { ...item };
```

Biome

[error] This let declares a variable that is only assigned once.

'updatedItem' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)